utils/scraper.ts

```diff
@@ -4,13 +4,11 @@ import axios, { AxiosResponse, CreateAxiosDefaults } from 'axios';
 import cheerio from 'cheerio';
 import { readFile, writeFile } from 'fs/promises';
 import HttpsProxyAgent from 'https-proxy-agent';
-import countries from './countries';
 
-type SearchResult = {
-   title: string,
-   url: string,
-   position: number,
-}
+import type { KeywordType, SettingsType, SearchResult, SerpApiSearchResults } from '../types.d';
+
+import { ScraperTypes } from '../types.d';
```

Comment:
Nextjs automatically imports types from the `types.d` file. No need to import the types .